wharehauora-wifi-gateway/wharehauora-wifi-gateway.ino

```diff
@@ -4,53 +4,73 @@
 #include <ESP8266WiFi.h>
 #define AP_NAME "WhareSensor"
```

Comment:
What happens if there is an ssid collision? Pretty unlikely I guess, but maybe add the mac address to the ssid? Probably just nitpicking....